aioshelly/rpc_device/device.py

```diff
@@ -538,3 +545,34 @@ def _parse_dynamic_components(self, components: dict[str, Any]) -> None:
                 for item in self._dynamic_components
             }
         )
+
+    async def _retrieve_blutrv_components(self, components: dict[str, Any]) -> None:
+        """Retrieve BLU TRV components."""
+        if self.model != MODEL_BLU_GATEWAY_GEN3:
+            return
+
+        if not self._config or not self._status:
+            raise NotInitialized
+
+        for component in components.get("components", []):
+            _key = component["key"].split(":")
+            if _key[0] == BLU_TRV_IDENTIFIER:
```

Comment:
Maybe?
```suggestion
            if _key[0] != BLU_TRV_IDENTIFIER:
                continue
```

Reply:
Why ? We need to make the call only for "blutrv" so only if the key is present

Reply:
Less indentation for subsequent lines